### ISSUE TITLE
Bug: _BR_T_SHOW_SEND_ENV_VOL_SEL_TRACK 

### ISSUE THREAD

**Issue by teeramusic** [reaper-oss/sws#1333]
Send levels become -inf when there is no user-programmed automation.

How to reproduce:
1) Add a few tracks
2) Send from the first track to the other tracks
3) execute _BR_T_SHOW_SEND_ENV_VOL_SEL_TRACK
4) execute _BR_T_SHOW_SEND_ENV_VOL_SEL_TRACK
5) execute _BR_T_SHOW_SEND_ENV_VOL_SEL_TRACK

Now all of the sends are -inf

Thanks for making this awesome extension!

**Comment by cfillion** [comment]
Is "When adding volume/pan envelopes, apply trim to envelope and reset trim" enabled? If so, it's the expected behavior. https://github.com/reaper-oss/sws/issues/1183#issuecomment-532051845

**Comment by teeramusic** [comment]
Yes I have that option checked set to "Always" but if trim is at unity, why should the envelope become -inf? Why should showing / hiding the envelopes change the level at all? It doesn't make any sense to me. If I click on the trim icon and the checkbox to make the envelope visible,  I can toggle it as many times as I want and there isn't any change to my envelope, it simply makes it visible or invisible? (as the name implies)
